packages/sso-button-react/package.json

```diff
@@ -18,6 +18,16 @@
   "files": [
     "dist"
   ],
+  "//": "./dist/** imports kept for backwards compatibility",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "./style.css": "./dist/style.css",
+    "./dist/style.css": "./dist/style.css",
```

Comment:
I'd not worry about backwards compatibility - worst case after update the dev would get a compilation error and they'd fix it, it's not a breaking change that would result in runtime bugs, unless the project has some very poor build system